src/mscorlib/src/System/String.cs

```diff
@@ -219,13 +218,19 @@ public sealed class String : IComparable, ICloneable, IConvertible, IEnumerable
             if (count == 0) {
                 return String.Empty;
             }
-            
+
+            if (count == 1) {
+                return value[startIndex] ?? String.Empty;
```

Comment:
Yes -- technically, you are indexing twice and if the indexer has side-effects you would execute them twice.

All right -- let's use the '??' operator for these :smile: